### ISSUE TITLE
Medusa doesn't detect a media

### ISSUE THREAD

**Issue by NicoLeOca** [pymedusa/SickRage#1088]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: last version
OS:

I face an issue and am now sure it's an issue.
I think it happened to me a few times with other shows but here is a example I am sure of

I came back a couple of minutes ago and found that:
![mrrobot episode missing](https://cloud.githubusercontent.com/assets/9487685/18418440/c8d13ae8-7846-11e6-96ea-f107be278b19.JPG)

but this morning the very episode was detected (I am sure because I had to force a rescan to have it detected).
![mrrobot folder content](https://cloud.githubusercontent.com/assets/9487685/18418448/f755a4e4-7846-11e6-9c34-8d58b1944bbc.JPG)

So Medusa tries to download it again.
as you see there are 2 different files for this episode in the folder, Medusa has tried to download it several times already.

here is my full log since this morning:
https://gist.github.com/NicoLeOca/d24a372168d5476c4d013c0b6b37ca81

Nzbget tells me it downloaded an episode again 3 hours ago; it is at least that old.

[EDIT] forcing a rescan detect the episode already in the folder
same as this morning

**Comment by NicoLeOca** [comment]
@fernandog @ratoaq2 
Guys, I am having trouble with this one lately.
can you please help to fix it?

S01E06 file isn't detected properly for some reasons.
Medusa download it again and again
![nzbget](https://cloud.githubusercontent.com/assets/9487685/18576133/ea0e7da6-7bde-11e6-9e28-eb084f81550f.JPG)
but a file is already in the proper folder (as displayed in the previous message above).

so this morning, I deleted the show from Medusa (without removing the files), restarted Medusa and added Mr Robot again.
While adding I refresh the main page of the show.
the whole season 1 is detected properly and the detection of the season 2 is pending and not detected yet:
![s01e06](https://cloud.githubusercontent.com/assets/9487685/18576238/fdab133c-7bdf-11e6-8c13-56d75422e092.JPG)


I refresh again and now I get that:
![s02e06](https://cloud.githubusercontent.com/assets/9487685/18576196/9cc1ebfe-7bdf-11e6-9cfe-a8a9e0793eec.JPG)

2 days ago (as I was talking with Fernandog about it on IRC) I archived the whole season 1.
However, Medusa tried to download S01E06 again this morning.

to summarize,
Medusa doesn't consider "archived"; it does detect my file but not consistantly. Sometimes it tries to redownload it, sometimes it considers S02E06 as S01E06

here are fresh logs if you have some time to investigate
https://gist.github.com/NicoLeOca/44f4778dfc956a7abab22b4147a77e5a

**Comment by ratoaq2** [comment]
It seems the problem is parsing the release name

```
2016-09-16 07:21:33 DEBUG    SHOWQUEUE-ADD :: [90eeff4] Parsed Mr.Robot.S02E06.eps2.4.m4ster-s1ave.aes.1080p.AMZN.WEBRip.DD5.1.x264-NTb into title: Mr Robot, season: None, episode: [6], episode_title: eps2 4 m4ster, screen_size: 1080p, format: WEBRip, audio_codec: DolbyDigital, audio_channels: 5.1, video_codec: h264, release_group: NTb, type: episode, absolute_episode: [], quality: 1080p WEB-DL
```

**Comment by ratoaq2** [comment]
Mr.  Robot has all these fancy episode titles that looks like a file name.
Season is not detected probably because of the s1ave word in episode title. 

I'll investigate it tonight what we can do about it

**Comment by ratoaq2** [comment]
The fix for this issue is divided into 2 parts:

- That release is being detected with 2 episodes and we were not properly discarding it, medusa was considering season 1. #1132  fixes that and it will be merged to develop any time soon.
- Only after releasing version 0.1.5 we're going to update guessit and the newest version parses that release properly: #1100

**Comment by NicoLeOca** [comment]
@ratoaq2 
you're a chief!! thank you very much!

**Comment by NicoLeOca** [comment]
Updated to 10b359e402376b54b12f8823680b5522ddd83c43

Mr. Robot S01E06 detected properly after a full show refresh

S02E06 isn't detected but I guess it is what you expect

**Comment by NicoLeOca** [comment]
@ratoaq2 
fully fixed now!  thank you!

**Comment by ratoaq2** [comment]
Great ☺️
